script.js

```diff
@@ -0,0 +1,41 @@
+const result = document.querySelector('#app')
+const ul = document.getElementById('#results')
+
+result.addEventListener('submit', () => {
+    fetch('http://localhost:3000/results')
+    .then(resp => resp.json())
+    .then(data => data.forEach((item) => {
+        const li = document.createElement('li')
+        li.textContent = item.fact
+        console.log(li)
+        ul.appendChild(li)
```

Comment:
The append doesn't work because of line 2, once you fix that the code should be able to identify where the variable ul is